app/cmd/root.go

```diff
@@ -60,6 +62,7 @@ func init() {
 	rootCmd.PersistentFlags().StringVarP(&rootOptions.Jenkins, "jenkins", "j", "", "Select a Jenkins server for this time")
 	rootCmd.PersistentFlags().BoolVarP(&rootOptions.Version, "version", "v", false, "Print the version of Jenkins CLI")
 	rootCmd.PersistentFlags().BoolVarP(&rootOptions.Debug, "debug", "", false, "Print the output into debug.html")
+	rootCmd.PersistentFlags().StringVarP(&rootOptions.Language, "language", "l", "", "Set HTTP Header Accept Language")
```

Comment:
I prefer to add this as a field of [type Config struct](https://github.com/jenkins-zh/jenkins-cli/blob/master/app/cmd/config.go#L68) instead of a global option.